src/ComputeSharp.D2D1/Intrinsics/D2D.cs

```diff
@@ -17,6 +19,7 @@ public static class D2D
     /// <returns>The color from the target input at the current coordinate, in <c>INPUTN</c> format.</returns>
     /// <remarks>This method is only available for simple inputs.</remarks>
     [HlslIntrinsicName("D2DGetInput")]
+    [HlslD2DIntrinsicInputType(D2D1PixelShaderInputType.Simple)]
```

Comment:
As per conversation on Discord, this should not produce an error when used with complex inputs. Still warrants a chat w/ D2D team, and a fix to the docs for `D2DGetInput()`.